File: CS-Examples/21_Fields/FormFieldsProperties.cs
using System;
using System.Drawing;
using System.IO;
using System.Windows.Forms;
using System.Xml.XPath;
using Spire.Doc;
using Spire.Doc.Documents;
using Spire.Doc.Fields;
using Spire.Doc.Collections;
using System.Text;

namespace FormFieldsProperties
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
           
			// Load the document from a file
			Document document = new Document(@"..\..\..\..\..\..\Data\FillFormField.doc");

			// Get the first section of the document
			Section section = document.Sections[0];

			// Get the second form field in the section
			FormField formField = section.Body.FormFields[1];

			// Check if the form field is a text input field
			if (formField.Type == FieldType.FieldFormTextInput)
			{
				// Set the text of the form field
				formField.Text = "My name is " + formField.Name;

				// Customize the text formatting of the form field
				formField.CharacterFormat.TextColor = Color.Red;
				formField.CharacterFormat.Italic = true;
			}

			// Save the modified document to a file
			document.SaveToFile("result.docx", FileFormat.Docx);

			// Dispose the document object
			document.Dispose();
			
            //Launch result file
            WordDocViewer("result.docx");

        }

        private void WordDocViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }

    }
}

File: VB-Examples/21_Fields/FormFieldsProperties.vb
Imports System.IO
Imports System.Xml.XPath
Imports Spire.Doc
Imports Spire.Doc.Documents
Imports Spire.Doc.Fields
Imports Spire.Doc.Collections
Imports System.Text

Namespace FormFieldsProperties
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			' Load the document from a file
			Dim document As New Document("..\..\..\..\..\..\Data\FillFormField.doc")

			' Get the first section of the document
			Dim section As Section = document.Sections(0)

			' Get the second form field in the section
			Dim formField As FormField = section.Body.FormFields(1)

			' Check if the form field is a text input field
			If formField.Type = FieldType.FieldFormTextInput Then
				' Set the text of the form field
				formField.Text = "My name is " & formField.Name

				' Customize the text formatting of the form field
				formField.CharacterFormat.TextColor = Color.Red
				formField.CharacterFormat.Italic = True
			End If

			' Save the modified document to a file
			document.SaveToFile("result.docx", FileFormat.Docx)

			' Dispose the document object
			document.Dispose()
			
			'Launch result file
			WordDocViewer("result.docx")

		End Sub

		Private Sub WordDocViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub

	End Class
End Namespace
